oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/AccessTokenService.java

```diff
@@ -168,7 +169,7 @@ private List<String> getIdentityClaims(JWTClaimsSet claimsSet)
             throws com.nimbusds.oauth2.sdk.ParseException, AccessTokenException {
         if (Objects.isNull(claimsSet.getClaim("claims"))) {
             LOG.info("No identity claims in AccessToken");
-            return null;
+            return Collections.emptyList();
```

Comment:
Why this change?

Reply:
SonarLint doesn't like it due to unclear intentionality:
`Returning null instead of an actual array, collection or map forces callers of the method to explicitly test for nullity, making them more complex and less readable.
Moreover, in many cases, null is used as a synonym for empty.`

Reply:
Am putting this back to null 👍